articles/api/management/v2/get-access-tokens-for-spas.md

```diff
@@ -1,32 +1,32 @@
 ---
-title: Get Management API Tokens for Single-Page Applications
-description: Learn how to get a specialized Access Token that will allow you to call Auth0's Management API endpoints. For use with Single-Page Applications (SPAs).
+description: Describes available scopes and endpoints for Management API tokens for Single-page Applications (SPAs).
 section: apis
-toc: true
 topics:
   - apis
   - management-api
   - tokens
 contentType: 
-    - how-to
+    - reference
 useCase: invoke-api
 ---
 
-# Get Management API Tokens for Single-Page Applications
+# Scopes and Endpoints for Management API Tokens
```

Comment:
The new title doesn't make sense to me either... The doc still only talks about SPAs but we have removed the SPA from title??

Reply:
Changed back to **Get Management API Tokens for Single-page Applications**